python/cuml/test/test_metrics.py

```diff
@@ -940,7 +940,7 @@ def test_pairwise_distances_sklearn_comparison(metric: str, matrix_size):
     Y = rng.random_sample(matrix_size)
 
     # For fp64, compare at 10 decimals, (5 places less than the ~15 max)
-    compare_precision = 10
+    compare_precision = 7
```

Comment:
It seems we need this changes asap that https://github.com/rapidsai/raft/pull/232 has been merged, @mdoijade can you update the PR to remove the change in get_raft so we can merge?